### PR TITLE
Expose info modal to extension

### PR DESCRIPTION
**Pull request by shubham251972**
**Notes for Reviewers**

This PR fixes #

This PR exposes info modal to extensions, to increase code reusability

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/Modals/Information/InfoModal.js [hunk 1]
```diff
@@ -432,7 +432,7 @@ const InfoModal_ = React.memo((props) => {
               onClick={handlePublishController}
               className={classes.copyButton}
               disabled={
-                isAdmin
+                isAdmin && !isPublished
```
Comment (leecalcote) [inline ui/components/Modals/Information/InfoModal.js:435]: @shubham251972 why is "isAdmin" being used an not keys?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: ui/components/Modals/Information/InfoModal.js (hunk 1 above)
Comment (Yashsharma1911) [reply ui/components/Modals/Information/InfoModal.js:435]: Ohh ho! no there is no use, we can now remove it as we are using keys, I'll remove it

Comment (shubham251972) [reply ui/components/Modals/Information/InfoModal.js:435]: No, there is no use of it, I'll remove it

Comment (shubham251972) [reply ui/components/Modals/Information/InfoModal.js:435]: Keys are already being use, it's just we might have forget to remove this isAdmin

Comment (leecalcote) [reply ui/components/Modals/Information/InfoModal.js:435]: @shubham251972 changes made?

Comment (leecalcote) [reply ui/components/Modals/Information/InfoModal.js:435]: @shubham251972 changes made?

Comment (leecalcote) [inline ui/components/Modals/Information/InfoModal.js:435]:
```suggestion
                !isPublished
```

**Review by Yashsharma1911 — APPROVED** [review]

**Pull request merged**